internal/broker/testdata/TestIsAuthenticated/golden/error_when_mode_is_newpassword_and_id_token_is_not_set/data/provider_url/test-user@email.com/token.json

```diff
@@ -0,0 +1 @@
+Definitely an encrypted token
```

Comment:
Is this highlighted red only on my screen?

Reply:
In GitHub I also see it highlighted in red. Maybe because it's not valid JSON?

Reply:
Could be... But it should be happening to other files as well then, no?

Reply:
Looking at the git history, those files were created without the `.json` suffix in a69d5884391b867edcfdb6e2760adfe5371808ef. They were renamed in 524fb8216f6aef0781d3b0efea60f70cafa98d7c, but because they were only renamed, GitHub didn't show the content in the diff. So this might actually be the first PR which creates a new `token.json` file with non-JSON content.